### DISCUSSION TITLE
Help on Filter Dialog in Grid Page?

### DISCUSSION THREAD

**Discussion by LSNicholls** [serenity-is/Serenity#6740] [Help] [answered]
Hi folks,

Feeling more than usually stupid today, I have looked all over the source to try to understand how the little "Edit Filter" button gets into the Northwind Order Grid sample.  I see references to an "edit filter dialog" and also a "filter panel" in the source, but I just can't figure out where this is being invoked as an attribute, or how else it works.  

Can anyone point me in the right direction?  Thank you as always.

![image](https://user-images.githubusercontent.com/88156108/228421002-2dc2f71a-d926-4881-83dc-f41158dc7cc4.png)

**Comment by arunkumar12712** [comment, marked as answer]
@Serenity.Decorators.filterable()

**Comment by LSNicholls** [comment]
OMG.  How did I miss that?!?  Thank you @arunkumar12712 !!
